source/fuzz/transformation_replace_copy_object_with_store_load.cpp

```diff
@@ -77,15 +80,19 @@ bool TransformationReplaceCopyObjectWithStoreLoad::IsApplicable(
     return false;
   }
   // |message_.variable_storage_class| must be Private or Function.
-  return ((message_.variable_storage_class() == SpvStorageClassPrivate ||
-           message_.variable_storage_class() == SpvStorageClassFunction));
+  return (message_.variable_storage_class() == SpvStorageClassPrivate ||
+          message_.variable_storage_class() == SpvStorageClassFunction);
```

Comment:
In general I see no reason to write:

```
return (e);
```

instead of simply:

```
return e;
```

i.e., I suggest removing the outer parentheses.

Reply:
Done. Thanks for pointing this out.